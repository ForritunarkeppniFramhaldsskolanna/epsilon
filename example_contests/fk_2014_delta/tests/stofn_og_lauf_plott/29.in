87
81 80 80 66 82 82 70 84 79 69 70 77 74 80 81 63 86 89 88 87 78 78 92 67 70 79 82 72 91 89 71 89 76 88 80 86 83 86 74 79 83 80 73 84 65 81 79 85 77 95 81 84 74 65 70 77 90 86 79 76 60 79 88 80 72 71 80 82 85 83 81 80 77 90 84 83 79 80 91 80 83 89 80 72 84 78 78
