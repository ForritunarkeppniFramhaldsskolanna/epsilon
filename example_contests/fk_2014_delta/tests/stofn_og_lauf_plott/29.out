6 0355679
7 000011222344466777788889999999
8 00000000000111112222333334444455666678889999
9 001125
